pkg/apis/pravega/v1beta1/pravega.go

```diff
@@ -68,6 +68,86 @@ const (
 
 	// DefaultInfluxDBSecretMountDir is the default mountpath of influxdb secret
 	DefaultInfluxDBSecretMountDir = "/etc/influxdb-secret-volume"
+
+	// DefaultReadinessProbeInitialDelaySeconds is the default initial delay (in seconds)
+	// for the readiness probe
+	DefaultControllerReadinessProbeInitialDelaySeconds = 20
+
+	// DefaultReadinessProbePeriodSeconds is the default probe period (in seconds)
+	// for the readiness probe
+	DefaultControllerReadinessProbePeriodSeconds = 10
+
+	// DefaultReadinessProbeFailureThreshold is the default probe failure threshold
+	// for the readiness probe
+	DefaultControllerReadinessProbeFailureThreshold = 3
+
+	// DefaultReadinessProbeSuccessThreshold is the default probe success threshold
+	// for the readiness probe
+	DefaultControllerReadinessProbeSuccessThreshold = 3
+
+	// DefaultReadinessProbeTimeoutSeconds is the default probe timeout (in seconds)
+	// for the readiness probe
+	DefaultControllerReadinessProbeTimeoutSeconds = 60
+
+	// DefaultLivenessProbeInitialDelaySeconds is the default initial delay (in seconds)
+	// for the liveness probe
+	DefaultControllerLivenessProbeInitialDelaySeconds = 60
+
+	// DefaultLivenessProbePeriodSeconds is the default probe period (in seconds)
+	// for the liveness probe
+	DefaultControllerLivenessProbePeriodSeconds = 15
+
+	// DefaultLivenessProbeFailureThreshold is the default probe failure threshold
+	// for the liveness probe
+	DefaultControllerLivenessProbeFailureThreshold = 4
+
+	// DefaultLivenessProbeSuccessThreshold is the default probe success threshold
+	// for the liveness probe
+	DefaultControllerLivenessProbeSuccessThreshold = 1
+
+	// DefaultLivenessProbeTimeoutSeconds is the default probe timeout (in seconds)
+	// for the liveness probe
+	DefaultControllerLivenessProbeTimeoutSeconds = 5
```

Comment:
We might have an issue with this setting in K8s 1.20+ (it was not set previously, so the default of 1 s was used).
From the documentation: 
> Before Kubernetes 1.20, the field timeoutSeconds was not respected for exec probes: probes continued running indefinitely, even past their configured deadline, until a result was returned.

If we are looking to allow the failure threshold of `4` and the period of `15`, the timeout needs to be at least `60`.

>This defect was corrected in Kubernetes v1.20. You may have been relying on the previous behavior, even without realizing it, as the default timeout is 1 second.